Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth
# Generic Artifact Origins

#### Changelog:
- Added random origins for generic artifacts

#### Developer changelog:
- Partially reworked effects

#### How to test:
- Proofread [this spreadsheet](https://docs.google.com/spreadsheets/d/1FVVfKRCmTzPfiqoKvcfllqOhR613hpCzm2XJ8Gk1moU/edit#gid=1339305026)
- Check in-game generic artifacts descriptions
